pandas/io/json/_json.py

```diff
@@ -31,20 +32,23 @@
 
 TABLE_SCHEMA_VERSION = "0.20.0"
 
+Serializable = Union[int, str, List, Dict]
+
 
 # interface to/from
 def to_json(
     path_or_buf,
```

Comment:
I think this should be `FilePathOrBuffer` but was failing with

```sh
pandas/io/json/_json.py:97: error: Item "Path" of "Union[Path, IO[Any]]" has no attribute "write"
```

I think this is actually a problem with the return of `_stringify_path` since it doesn't indicate that Path objects get converted to strings. Can open an issue as a follow up but @simonjayhawkins may already be working on that

Reply:
yeah, should already be fixed with overloads in #27732.